activiti-cloud-runtime-bundle-service/activiti-cloud-starter-runtime-bundle/src/main/java/org/activiti/cloud/starter/rb/configuration/RuntimeBundleSwaggerConfig.java

```diff
@@ -52,9 +55,41 @@ public GroupedOpenApi runtimeBundleApi(@Value("${activiti.cloud.swagger.rb-base-
             .addOpenApiCustomizer(openApi ->
                 openApi.addExtension(BaseOpenApiBuilder.SERVICE_URL_PREFIX, swaggerBasePath)
             )
+            .addOpenApiCustomizer(openApiCustomizer())
             .build();
     }
 
+    public OpenApiCustomizer openApiCustomizer() {
+        return openAPI -> {
+            openAPI
+                .getPaths()
+                .values()
+                .forEach(val -> {
+                    val
+                        .readOperations()
+                        .forEach(operation -> {
+                            operation
+                                .getResponses()
+                                .forEach((key, value) -> {
```

Comment:
Is it possible to use map/flatMap instead of nested foreach?

Reply:
Done